physical_infra_provider_rest_api/usage/topics/get_providers.adoc

```diff
@@ -0,0 +1,27 @@
+=== Get Providers
+
+. Log in to CloudForms Management Engine API as a user who has permission to access the API. The default user is admin, password smartvm.
+
+. Use the following url structure for access the api **"<CLOUDFORMS ADDRESS>:<PORT>/api/" (ex. localhost:3000/api/)**.
+
+. Use the following request for get all provider’s resource. **(ex. localhost:3000/api/providers)**.
+
+. It’s returned a JSON with the following keys
+    
+    .. name **(URI name)**.
```

Comment:
Maybe you can put the keys into a table like you did in step 4 of section 1.5 (Get a Physical Server by ID).